Cargo.toml

```diff
@@ -16,6 +16,7 @@ license = "Apache-2.0"
 ctrlc = "3.1"
 http = "0.2"
 log = "0.4"
+macro_rules_attribute = "*"
```

Comment:
@alsuren  I'm hesitant to add this dependency. Perhaps instead we can implement this as a macro in `goose_codegen`, and generally add other macros in this dependency to improve goose ergonomics? (The code currently in there can be removed).

I don't like that it's a 0.0.1 release, that I don't see any other modules using it on crates.io, and that we have to invoke it before every load test function.